modules/dev-server/main.tf

```diff
@@ -401,7 +401,7 @@ exec consul agent -server \
   -hcl='cert_file = "/consul/${var.datacenter}-server-consul-0.pem"' \
   -hcl='key_file = "/consul/${var.datacenter}-server-consul-0-key.pem"' \
   -hcl='auto_encrypt = {allow_tls = true}' \
-  -hcl='ports { https = 8501, grpc_tls = 8502 }' \
```

Comment:
This is here because I merged https://github.com/hashicorp/terraform-aws-consul-ecs/pull/138. But `grpc_tls` is not valid in 1.13.3